modules/appeals_api/app/services/appeals_api/central_mail_updater.rb

```diff
@@ -119,13 +110,15 @@ def update_appeal_status!(appeal, central_mail_status)
     def log_exception(e, appeal, status)
       details = {
         class: self.class.to_s,
+        detail: 'Error when trying to update appeal status',
         appeal_type: appeal.class.to_s,
         appeal_id: appeal.id,
         appeal_status: appeal.status,
         attempted_status: status
       }
-
-      log_exception_to_sentry e, details
+      error_details = { details: e.message }
+      error_details[:back_trace] = e.backtrace.join("\n") unless e.backtrace.nil?
```

Comment:
I confirmed that the `backtrace` is always `nil` here as well.

Reply:
changed to log the detailed info that we also send to slack